### ISSUE TITLE
Colorblind safe button colors

### ISSUE THREAD

**Issue by willcbaker** [18F/web-design-standards#1296]
The colors chosen for button states are difficult to discern.

most notably: color-secondary-X at [web-design-standards/src/stylesheets/core/_variables.scss](https://github.com/18F/web-design-standards/blob/259b7af5d428a36c8fcf9ed1aeb079d00f297f9a/src/stylesheets/core/_variables.scss)

on `usa-button-secondary` at [web-design-standards/src/stylesheets/elements/_buttons.scss](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/elements/_buttons.scss)
#811 #11605

**Comment by maya** [comment]
@willcbaker thanks so much for opening this issue. Can you give us a little more info about the problem you're experiencing? What did you expect to happen when viewing the button states? Is there a particular state that you thought was difficult to discern or what it all three states (default, hover, active)?

cc: @nickbristow for awareness

**Comment by willcbaker** [comment]
all three states are difficult to discern, definitely for secondary(red), primary (blue) is not as difficult but also not easy.
I personally notice a change HAS occurred, but I cannot determine which state I am in by purely looking at the screen (aka. if someone else where pressing the button and I was simply observing)

**Comment by maya** [comment]
To my knowledge, the WCAG 2.0 doesn't have guidelines that each state state must be discernible by a certain amount, it's more of an affordance. The idea is to know a change has occurred which you mentioned being able to see so closing for now.
